src/style.css

```
@@ -1 +1,9 @@
 /* styles go here */
+fieldset {
+  padding-bottom: 0;
+  margin-bottom: 20px;
+}
+
+div.form-field {
```

`div` selector here is redundant, that's just increasing specificity
```suggestion
.form-field {
```